Jenkinsfile

```diff
@@ -64,7 +76,7 @@ pipeline{
           export GRADLE_USER_HOME=$WORKSPACE/$GRADLE_DIR
           export PATH=$GRADLE_USER_HOME:$JAVA_HOME/bin:$PATH
           cd java-client-api
-          ./gradlew mlDeploy
+          ./gradlew mlDeploy -PmlForestDataDirectory=/space
```

Comment:
I added this in a separate PR where was I fixing a config issue with the functional tests